core/src/Model/Enemy.java

```diff
@@ -1,17 +1,11 @@
 package Model;
 
 public abstract class Enemy extends Entity{
-    private float damage;
-    private String imagePath = ""; // hur gör man att de måste finnas en?
+    private double damage;
+    private String enemyName = ""; // Should be the same as its corresponding prefix under assets.
```

Comment:
Is both enemyName and entityName needed?

Reply:
yes entityName = name of species 
enemyName = individual name of the enemy